src/main/java/org/springframework/retry/annotation/Backoff.java

```diff
@@ -130,7 +131,10 @@
 	 * this to true to have the backoff delays randomized, so that the maximum delay is
 	 * multiplier times the previous delay and the distribution is uniform between the two
 	 * values. Use {@code #{...}} for one-time evaluation during initialization, omit the
-	 * delimiters for evaluation at runtime.
+	 * delimiters for evaluation at runtime. This expression is evaluated at configuration
+	 * phase. If it returns true and ({@link #multiplier()} &gt; 1.0) then
+	 * {@link org.springframework.retry.backoff.ExponentialRandomBackOffPolicy} is created
+	 * else {@link org.springframework.retry.backoff.ExponentialBackOffPolicy} is created
```

Comment:
The assumption about `multiplier` is not correct: it simply could be resolved at runtime as we discuss with you in other place.
So, consider to revise this one in exactly what we want to say: unlike other expressions, this one is always evaluated at configuration phase, so the framework can determine to build `ExponentialRandomBackOffPolicy` or `ExponentialBackOffPolicy`.

Reply:
This is fixed. Please check